### DISCUSSION TITLE
What's the fastest way to iterate on prompts using AutoGen?

### DISCUSSION THREAD

**Discussion by philsalesses** [microsoft/autogen#733] [General]
We were looking at using Vellum.ai for something we were planning to build but when we learned about AutoGen we started exploring. We're almost sold on it, but we have no way to iterate on prompts or get metric data on prompts it seems. What is the best practices for this part?

**Comment by rickyloynd-microsoft** [comment]
@philsalesses Sounds interesting. Can you give more details about the kind of prompt iteration and metric data you have in mind?

**Comment by sonichi** [comment]
I assume this question is related to evaluation and if so, https://microsoft.github.io/autogen/blog/2023/11/20/AgentEval is worth checking.
